### PR TITLE
ISSD-2926 [Testray] Loading indefinitely when creating a task

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

6 files changed, 160 insertions(+), 166 deletions(-)

**Comment by rafaelfs98** [comment]
ci:test:sf

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx [hunk 1]
```diff
@@ -45,26 +39,25 @@ export type RendererFields = {
...
 	form,
 	onChange,
-	setFieldOptions,
 }) => {
 	const params = useParams();
 
 	const [fieldDisabled, setFieldDisabled] = useState({});
-	const [isLoading, setIsLoading] = useState(true);
+
+	// const [isLoading, setIsLoading] = useState(true);
```
Comment (claricebrasil) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx:60]:
```suggestion

```

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx [hunk 2]
```diff
@@ -97,36 +83,40 @@ const Renderer: React.FC<RendererProps> = ({
...
 
-	const fetchResources = useCallback(async () => {
-		const parameters = safeJSONParse(paramsMemoized);
-
-		const fieldsWithResource = fieldsMemoized.filter(
-			({resource}) => resource
-		);
-
-		const _fieldOptions: any = {};
+	const {data: filderOptions = {}, isLoading} = useSWR(
```
Comment (claricebrasil) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx:86]:
```suggestion
	const {data: filterOptions = {}, isLoading} = useSWR(
```

Comment (claricebrasil) [reply workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx:86]: Or
```suggestion
	const {data: fieldOptions = {}, isLoading} = useSWR(
```

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx [hunk 3]
```diff
@@ -183,7 +173,7 @@ const Renderer: React.FC<RendererProps> = ({
 
 				const getOptions = () => {
 					const _options =
-						fieldOptions[name] ||
+						filderOptions[name] ||
```
Comment (claricebrasil) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx:176]:
```suggestion
						fieldOptions[name] ||
```

Comment (rafaelfs98) [reply workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx]: Resolved in  [[ISSD-2926 Rename](https://github.com/is-solutions-delivery/liferay-portal/pull/5504/commits/bcfa0cdf314ff4dec650821c669f85bc517c8974)]

**Pull request closed without merging**